src/harness/harnessLanguageService.ts

```diff
@@ -336,6 +336,9 @@ module Harness.LanguageService {
         getOccurrencesAtPosition(fileName: string, position: number): ts.ReferenceEntry[] {
             return unwrapJSONCallResult(this.shim.getOccurrencesAtPosition(fileName, position));
         }
+        getDocumentHighlights(fileName: string, position: number, filesToSearch: string[]): ts.DocumentHighlights[] {

```

Comment:
I'm following the name of Roslyn here.  This is a general mechanism for highlighting, it doesn't have to be for references.